dotnet/src/Connectors/Connectors.Memory.Redis/RedisVectorStoreCollectionCreateMapping.cs

```diff
@@ -177,6 +177,7 @@ public static string GetSDKDistanceAlgorithm(VectorStoreRecordVectorProperty vec
         return vectorProperty.DistanceFunction switch
         {
             DistanceFunction.CosineSimilarity => "COSINE",
+            DistanceFunction.CosineDistance => "COSINE",
```

Comment:
These are both the same value, is this correct?

Reply:
Yes, redis only support distance, but I am mapping back to similarity if needed on response.